crates/agglayer-config/src/epoch.rs

```diff
@@ -65,7 +48,7 @@ mod tests {
         let epoch = Epoch::TimeClock(TimeClockConfig::default());
         let serialized = serde_json::to_string(&epoch).unwrap();
 
-        assert_eq!(serialized, r#"{"time-clock":{"epoch-duration":5}}"#);
+        assert_eq!(serialized, r#"{"time-clock":{"epoch-duration":"5s"}}"#);
```

Comment:
How does it deal with not having the `s` ?
Do we have a fallback for config that would use the previous format?

Reply:
> How does it deal with not having the `s` ? Do we have a fallback for config that would use the previous format?

Yes the old config still loads fine. You can choose either to use string with units, like `"5s"` or `"15h"`, or just a json number which is treated as number of seconds, same as with `DurationSeconds`.